website/pages/blog/announcing-conductor-alpha-release.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: 'Announcing Conductor, a robust, free MIT-Licensed GraphQL Gateway'
+tags: [graphQL, conductor, gateway, alpha]
+authors: [Yassin Eldeeb]
+date: 2024-01-15
+description:
+  Announcing the alpha release of Conductor, an MIT-licensed, open-source GraphQL Gateway. Designed
+  for simplicity and efficiency, it's a robust solution for modern GraphQL needs.
+image: /blog-assets/graphql-conductor-alpha-release/cover.png
+thumbnail: /blog-assets/graphql-conductor-alpha-release/thumbnail.png
+---
+
+After months of intense development, we're thrilled to announce the public launch of
+[Conductor](https://the-guild.dev/graphql/gateway) - our latest venture into the world of GraphQL!
+
+## For the Community, Embracing All GraphQL Projects
+
+Conductor has been crafted with a clear vision: to empower developers with their advanced GraphQL
+needs. Here's what sets it apart:
+
+- **Built with Rust**: Focused on performance and reliability, leveraging Rust's safety and
+  concurrency capabilities.
+- **Exceptional Developer Experience**: We understand that developer experience is paramount. That's
+  why Conductor is designed to be intuitive, straightforward, and hassle-free, allowing you to focus
+  on what matters most - building great and performant GraphQL applications.
+- **Real open-source**: Conductor is designed to be accessible to everyone, it is open-source (MIT)
```

Comment:
can we include here also that the plugins are open source?
especially the ones that are being offered on Enterprise plans on other gateways like JWT, etc...?

Reply:
sure!